### ISSUE TITLE
Add infrastructure for upvote/downvote reports or to flag inappropriate content

### ISSUE THREAD

**Issue by asbarve** [urbanriskmap/riskmap#209]
~1 Add report inappropriate button to reports in front end~
2 Extend API to add attribute 'disabled'
3 develop simple front end with login for URL to select and take down the report. 

Get UROP to do 2+3 ?

**Comment by talltom** [comment]
@asbarve can we prefil the subject line and the body of the email message? Specifically this should include a link to the report in question and its id, e.g. "Report /map/jakarta/124"

**Comment by talltom** [comment]
Couple of UI feedback items from prototype:
- thumbs up/down should be toggle on/off style buttons as per the share and flag buttons
- remove blue box which appears on click
- first click of either up or down vote is not responsive (waiting for callback?). Suggest updating UI at same time call is made - this is a non critical process (less of a problem if it fails).
- should we align the wrist of the hands?

**Comment by matthewberryman** [comment]
Just noting a bug in that I can repeatedly click on thumbs up / thumbs down after it's already highlighted to add however down and upvotes I want. You could add check to see if it's already highlighted, although that could be worked around by someone refreshing the page or debugging to find out the HTTP request made and then repeatedly calling that using curl/postman, etc., so perhaps a better solution is to use cookies.

**Comment by talltom** [comment]
@matthewberryman thanks. @asbarve - isn't this the design?

**Comment by asbarve** [comment]
1. @talltom - it was a bug! @matthewberryman thanks for catching that we have fixed it now.
2. @matthewberryman: yes it is possible to find out HTTP request from debugging, but since we don't use upvote/downvote for any analytics we might try it out and see how people interact with it, meanwhile, @ojha-url and I will look into using cookies.
3. @talltom we have improved the responsiveness of click event
